cmake/deps.txt

```diff
@@ -23,7 +23,7 @@ microsoft_gsl;https://github.com/microsoft/GSL/archive/refs/tags/v4.0.0.zip;cf36
 microsoft_wil;https://github.com/microsoft/wil/archive/5f4caba4e7a9017816e47becdd918fcc872039ba.zip;fd119887d0d17c37adf1fc227b054befa28158ad
 mimalloc;https://github.com/microsoft/mimalloc/archive/refs/tags/v2.1.1.zip;d5ee7d34223d0567892db5179849939c8769dc41
 mp11;https://github.com/boostorg/mp11/archive/refs/tags/boost-1.79.0.zip;c8f04e378535ededbe5af52c8f969d2dedbe73d5
-onnx;https://github.com/onnx/onnx/archive/3b58938e025c41d2fcd89fa22028eefaa81a18ad.zip;e0e5dda9eea5cd5ecae3bd8be86e477016b6be02
```

Comment:
NIP: I see this commit is not on ONNX 1.14 branch. Eventually commit here shall come from ONNX rel 1.14. Please make sure in ONNX mark PRs that are needed by 1.14 release so once this commit is switched to ONNX rel 1.14 there is no changes missing.

Reply:
For this upcoming ORT release, we cannot include the things that will not be in ONNX 1.14 release.